example.json

```diff
@@ -1,18 +1,18 @@
 {
     "variables": {
-        "sdc_url": "{{env `SDC_URL`}}",
-        "sdc_account": "{{env `SDC_ACCOUNT`}}",
-        "sdc_key_id": "{{env `SDC_KEY_ID`}}",
-        "sdc_key_path": "{{env `SDC_KEY_PATH`}}"
+        "triton_url": "{{env `TRITON_URL`}}",
```

Comment:
It is probably preferable to use the `SDC_*` vars. For context, node-sdc uses the following:

- `SDC_URL`
- `SDC_ACCOUNT`
- `SDC_KEY_ID`

[node-triton](https://www.joyent.com/blog/introducing-the-triton-command-line-tool) will read those env vars as well as set them [from a profile](https://www.joyent.com/blog/introducing-the-triton-command-line-tool#using-profiles) with `triton env`, which is expected to be called as `eval "$(triton env)`. 

```bash
$ triton env
export SDC_URL="https://us-sw-1.api.joyent.com"
export SDC_ACCOUNT="<my account>"
unset SDC_USER
export SDC_KEY_ID="SHA256:<my key>"
unset SDC_TESTING
export TRITON_PROFILE="us-sw-1"
# Run this command to configure your shell:
#     eval "$(triton env -st us-sw-1)"
```

I'm not sure if that means we should use `TRITON_KEY_MATERIAL` or `SDC_KEY_MATERIAL`, however.